### PR TITLE
First impression, update the look of the landing page

### PR DESCRIPTION
**Pull request by juggernautsei**
Fixes #

#### Short description of what this resolves:
Resolves the plain look of the front landing page. We are busy modernizing the back end. The front end can use a little TLC. This little bit of CSS can go along way towards the initial impression of the software, in my humble opinion. 

#### Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by tywrenn** [comment]
@juggernautsei stylelint check failed here because you don't need to have `-webkit` CSS added with it as it is automatically added with our autoprefixer plugin. Also it failed because `border-radius: 10px 10px 10px 10px` can be condensed to `border-radius: 10px`.

Also I recommend changing the box shadow CSS to this SCSS equivalent:
`box-shadow: 0 30px 60px 0 rgba($black, 0.3);`

**Comment by juggernautsei** [comment]
https://www.w3schools.com/sass/sass_functions_color.asp
https://sass-lang.com/documentation/modules/color

https://sass-lang.com/documentation/variables

@tywrenn, I did what you suggested by when I tried it in my browser. It did not produce the box-shadow.  So, I did some research. The code suggestion you made would be best used in the login.scss file. There, I did not get the error messages that I was receiving in the login.css file. But it seems for right now the login.css file is controlling the styling on the landing page and not the login.scss. Correct me if I am wrong.  Above are my sources.

**Comment by tywrenn** [comment]
@juggernautsei just saw ur revision -- the `black` attribute needs to have the $ symbol before it to call the SASS/SCSS variable

**Comment by tywrenn** [comment]
The SCSS file converts into the CSS file when you compile OpenEMR which is done through gulp

**Comment by tywrenn** [comment]
interface/themes/login_page.scss
 32:3   ✖  Expected background to come      order/properties-alphabetical-order
           before border-radius                                                
 36:3   ✖  Expected box-shadow to come      order/properties-alphabetical-order
           before padding                                                      
 36:34  ✖  Unexpected named color "black"   color-named                        
 36:39  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function                                           
interface/themes/colors/openemr5/login.scss
 11:3   ✖  Expected $-variable to come      order/order                        
           before declaration                                                  
 12:3   ✖  Expected border-radius to come   order/properties-alphabetical-order
           before max-width                                                    
 13:40  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function

File: interface/themes/login_page.scss [hunk 1]
```diff
@@ -28,14 +28,12 @@ body.login {
...
-  -webkit-border-radius: 10px 10px 10px 10px;
-  border-radius: 10px 10px 10px 10px;
+  border-radius: 10px;
   background: $white;
   margin-top: 75px;
   max-width: 785px;
   padding: 25px;
-  -webkit-box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
-  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
+  box-shadow: 0 30px 60px 0 rgba(black,0.3);
```
Comment (tywrenn) [inline interface/themes/login_page.scss:36]: This should be $black

File: interface/themes/colors/openemr5/login.scss [hunk 2]
```diff
@@ -8,6 +8,9 @@
   border-top: solid $dark 40px;
   margin-top: 5%;
   max-width: 500px;
+  $black: #000 !default;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
```
Comment (tywrenn) [inline interface/themes/colors/openemr5/login.scss:13]: $black shouldn't be defined here as that variable gets imported from `color_base.scss`

Comment (tywrenn) [reply interface/themes/colors/openemr5/login.scss:13]: SCSS has different files that create an object-oriented structure that eventually gets compiled into one CSS file. If you look around, `@import`'s show which files go into which files which go into one big file.

File: interface/themes/login_page.scss (hunk 1 above)
Comment (juggernautsei) [reply interface/themes/login_page.scss:36]: I have tried it with and without the $. And neither of them work in the login.css. I tried all the major browsers and I do keep my browsers up to date.

File: interface/themes/colors/openemr5/login.scss (hunk 2 above)
Comment (juggernautsei) [reply interface/themes/colors/openemr5/login.scss:13]: That makes perfect sense. Here are the new changes.

Comment (tywrenn) [reply interface/themes/colors/openemr5/login.scss:13]: @juggernautsei The above code that I mentioned still hasn't been changed

File: interface/themes/colors/openemr5/login.scss [hunk 3]
```diff
@@ -8,7 +8,6 @@
   border-top: solid $dark 40px;
   margin-top: 5%;
   max-width: 500px;
-  $black: #000 !default;
```
Comment (juggernautsei) [inline interface/themes/colors/openemr5/login.scss]: This line has been removed

File: interface/themes/login_page.scss [hunk 4]
```diff
@@ -33,7 +33,7 @@ body.login {
   margin-top: 75px;
   max-width: 785px;
   padding: 25px;
-  box-shadow: 0 30px 60px 0 rgba($black,0.3);
```
Comment (juggernautsei) [inline interface/themes/login_page.scss]: this line has been removed. Now I have to put it back. arrrga!

File: interface/themes/login_page.scss [hunk 5]
```diff
@@ -33,7 +33,7 @@ body.login {
   margin-top: 75px;
   max-width: 785px;
   padding: 25px;
-  box-shadow: 0 30px 60px 0 rgba(0,0,0,0.3);
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
```
Comment (juggernautsei) [inline interface/themes/login_page.scss:36]: removed the variable then put it back in. For my older code, we need to use the 0,0,0.

Comment (juggernautsei) [reply interface/themes/login_page.scss:36]: I just learned something new in GitHub. How to make the above kind of comment.

**Comment by tywrenn** [comment]
This is how it looks on my end:
image
Looks like the shadow is really really huge xD

**Comment by juggernautsei** [comment]
https://bootsnipp.com/snippets/dldxB
@tywrenn this is where the concept came from. The shadow box looks huge because the login form is huge. There is no reason for it to be that big other than to accommodate the logo which should go outside the login box like this https://athenanet.athenahealth.com/1/1/login.esp. The logo should not be inside the login form in my opinion. That is what makes is oversized. We can cut down the size of the login box by removing the logo. The shadow box gives the viewer a sense of modern coding. I have been talking with newcomers to the project and they think we should do something about the first impression. People push passed the landing page hoping there is something better inside and they are let down some more. I think we could attract more users if the program just looked better when the new prospects land on the page. 
Mines look like this:

image

**Comment by tywrenn** [comment]
@juggernautsei Good point -- I like this idea. I'll follow up this pull with another one that does a drastic change to the login form that cuts it down a bit and makes it more simple. In the mean time, please fix the following so Travis CI can pass:
interface/themes/login_page.scss
 32:3   ✖  Expected background to come      order/properties-alphabetical-order
           before border-radius                                                
 36:3   ✖  Expected box-shadow to come      order/properties-alphabetical-order
           before padding                                                      
 36:40  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function                                           
interface/themes/colors/openemr5/login.scss
 11:3   ✖  Expected border-radius to come   order/properties-alphabetical-order
           before max-width                                                    
 12:40  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function

**Comment by tywrenn** [comment]
@juggernautsei Still failed :(

interface/themes/colors/openemr5/login.scss
 11:3   ✖  Expected border-radius to come   order/properties-alphabetical-order
           before max-width                                                    
 12:40  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function

File: interface/themes/login_page.scss [hunk 6]
```diff
@@ -28,12 +28,12 @@ body.login {
 }
 
 .login .container {
-  border-radius: 10px;
   background: $white;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black, 0.3);
   margin-top: 75px;
```
Comment (juggernautsei) [inline interface/themes/login_page.scss:34]: Something is wrong with Travis. Could you please explain how what Travis is reporting? 

image

File: interface/themes/colors/openemr5/login.scss [hunk 7]
```diff
@@ -8,6 +8,8 @@
   border-top: solid $dark 40px;
   margin-top: 5%;
   max-width: 500px;
+  border-radius: 10px;
+  box-shadow: 0 30px 60px 0 rgba($black,0.3);
 }
```
Comment (stephenwaite) [inline interface/themes/colors/openemr5/login.scss:13]: hi @juggernautsei , travis is barking at this file not the one below :)

Comment (tywrenn) [reply interface/themes/colors/openemr5/login.scss:13]: @juggernautsei Yeah Travis is referring to this file not the one you're looking at

**Comment by tywrenn** [comment]
@juggernautsei 

interface/themes/colors/openemr5/login.scss
  9:3   ✖  Expected border-radius to come   order/properties-alphabetical-order
           before border-top                                                   
 10:40  ✖  Expected single space after ","  function-comma-space-after         
           in a single-line function

**Comment by tywrenn** [comment]
@bradymiller Can u run stylelint fix on his branch for me 😂

**Comment by tywrenn** [comment]
@bradymiller ready for merge

**Comment by bradymiller** [comment]
hi @juggernautsei , Bringing this in now. Thanks for the improvement! -brady

**Pull request merged**